js/util/glossary/index.js

```diff
@@ -0,0 +1,22 @@
+import Glossary from 'glossary-panel';
+import terms from './terms';
+
+// Optional configurion objects
+const selectors = {};
+const classes = {};
+
+let glossary;
+function getGlossary() {
+  return glossary;
+}
+
+// Wait for DOM to be loaded before initializing the glossary
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Is this something we could potentially fold back into the glossary library? It would be nice to be able to improve that. :-/

Reply:
Mixed feelings... Generally I prefer libraries to not handle this kind of loading and leave it up to the app. I think it's strange that the glossary calls DOM APIs, because it makes it a bit more difficult to run in environments where the DOM is not available (like running tests in node). I would prefer to pass in DOM elements instead, which would allow the app to control when to re-initialize the glossary in case elements move around or something.